cmd/cmd.go

```diff
@@ -66,8 +66,8 @@ func init() {
 	// will be global for your application.
 
 	rootCmd.PersistentFlags().StringVar(&kubeConfig, "kubeconfig", "", "kubernetes config path")
-	rootCmd.PersistentFlags().StringVar(&sourceStorageClass, "sourcestoraageclass", "", "source storageclass from which all PVC need to be migrated")
-	rootCmd.PersistentFlags().StringVar(&destinationStorageClass, "destinationstorageclass", "", "destination storageclass (CSI storageclass) to which all PVC need to be migrated")
+	rootCmd.PersistentFlags().StringVar(&sourceStorageClass, "source-storageclass", "", "source storageclass from which all PVC need to be migrated")
```

Comment:
```suggestion
	rootCmd.PersistentFlags().StringVar(&sourceStorageClass, "source-sc", "", "source storageclass from which all PVC need to be migrated")
```
was thinking to make cli agr more concise. thoughts?

Reply:
@Yuggupta27 @Madhu-1 are we good?